### PR TITLE
CIV-10055 update hwf ref number

### PR DESCRIPTION
**Pull request by kalachandrasekar1**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-10055

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by ElifDenizEsea — DISMISSED** [review]

File: ccd-definition/CaseField/CaseField-HWF-R2-CUI-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "caseTitleWithHWFFeeNumber",
+    "Label": "### ${helpWithFees.helpWithFeesReferenceNumber} ${applicant1.partyName} Vs ${respondent1.partyName}",
```
Comment (portizdev) [inline ccd-definition/CaseField/CaseField-HWF-R2-CUI-nonprod.json:5]: Use `<h2 class=\"govuk-heading-1\">  ${helpWithFees.helpWithFeesReferenceNumber} ${applicant1.partyName} Vs ${respondent1.partyName}</h2>"` to be consistent with all the views. Same with caseTitleWithHWFHearingNumber

File: ccd-definition/CaseField/CaseField-HWF-R2-CUI-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "caseTitleWithHWFFeeNumber",
```
Comment (portizdev) [inline ccd-definition/CaseField/CaseField-HWF-R2-CUI-nonprod.json:4]: Use caseTitleWithHWFClaimFeeNumber to be consistent pls

File: ccd-definition/CaseEventToComplexTypes/UpdateReferenceNum-HelpWithFee-R2-CUI-nonprod.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    "FieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "ID": "HelpWithFeesDetails",
+    "CaseEventID": "UPDATE_HELP_WITH_FEE_NUMBER",
+    "CaseFieldID": "hwFeesDetails",
+    "ListElementCode": "hearingFee",
+    "EventElementLabel": "### Help with Fees type:  HwF Hearing Fee",
```
Comment (portizdev) [inline ccd-definition/CaseEventToComplexTypes/UpdateReferenceNum-HelpWithFee-R2-CUI-nonprod.json:17]: In other views we are not using `### `, I don't know if in your view it is required to appear in bold. Pls check.

File: ccd-definition/CaseField/CaseField-HWF-R2-CUI-nonprod.json (hunk 1 above)
Comment (kalachandrasekar1) [reply ccd-definition/CaseField/CaseField-HWF-R2-CUI-nonprod.json]: updated

File: ccd-definition/CaseField/CaseField-HWF-R2-CUI-nonprod.json (hunk 2 above)
Comment (kalachandrasekar1) [reply ccd-definition/CaseField/CaseField-HWF-R2-CUI-nonprod.json]: updated

**Review by jeswanth-hmcts — DISMISSED** [review]

**Review by ElifDenizEsea — APPROVED** [review]

**Review by jeswanth-hmcts — APPROVED** [review]

**Review by ElifDenizEsea — DISMISSED** [review]

**Review by ElifDenizEsea — DISMISSED** [review]

**Review by manish14836 — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Pull request merged**